core/src/main/java/nu/yona/server/exceptions/InvalidDataException.java

```diff
@@ -85,4 +85,9 @@ public static InvalidDataException excessEmailAddress()
 	{
 		return new InvalidDataException("error.user.email.address.not.supported");
 	}
+
+	public static InvalidDataException excessGoals()

```

Comment:
I think the current name shows the intent better. The method above, `excessEmailAddress`, has the same purpose and message ID.